libs/types/src/index.ts

```diff
@@ -16,5 +16,6 @@ export * from './printing';
 export * from './result';
 export * from './smartcard_auth';
 export * from './tallies';
+export * from './ui_theme';
```

Comment:
Any particular reason you put this in `libs/types`? It's nice to move types there but it all seems UI dependent. Just curious.

Reply:
No solid reason - I haven't fully developed an intuition around our code organization WRT types, but thought there could be a case where we want to reference theme types or the color enum from a package that we don't want depending on libs/ui (e.g. in the unlikely case, we reference a color or theme from an election config or something similar)